### ISSUE TITLE
create a contents.md file

### ISSUE THREAD

**Issue by blueforesticarus** [cougarTech2228/2016-Game-Code-Base#2]
Part of good documentation is having a  file which outlines what each of the files and folders in the project are for, so that someone who has no idea how the project is laid out can easily find where certain features are implemented. A contents.md file should be added explaining what each file/class is for and how the code is laid out as a whole.

**Comment by ZacharyDuBois** [comment]
Well, we should do that if code is not commented. We should be using code document engines like pydoc, javadoc, and phpdoc. This can be corrected off-season.

**Comment by blueforesticarus** [comment]
That is a decent solution, but also the lazy one. Auto generated documentation from source file comments is never as readable or straight forward as that which is formatted and compiled by an actual developer; it nice to have a single file which gives a logical overview of the project layout. Of course it is good to have the javadoc in addition.

**Comment by srvers12** [comment]
I agree with Zach!!!!!!!!!!! This is defined in the software handbook for
team 2228!!

**Comment by srvers12** [comment]
Also - the use of UML(Industry standard) which is in Eclispe should be
used. This will be added to the handbook shortly.
